dali/kernels/test/tensor_test_utils.h

```diff
@@ -65,28 +62,36 @@ void CheckEqual(
   }
 }
 
+
 template <int dim1, int dim2>
 void CheckEqual(const TensorListShape<dim1> &s1, const TensorListShape<dim2> &s2) {
   int max_errors = detail::DefaultMaxErrors;
   CheckEqual(s1, s2, max_errors);
 }
 
+
 template <typename T>
 T printable(T t) { return t; }
 
+
 inline int printable(char c) { return c; }
+
```

Comment:
Why? Please remove these extra lines - you can keep one between the functions if you insist, but two empty lines between one-liners is too much.

Reply:
Who says, that it's too much? :)